bundler/lib/bundler/templates/newgem/ext/newgem/extconf-c.rb.tt

```diff
@@ -2,4 +2,9 @@
 
 require "mkmf"
 
+# Makes all symbols private by default to avoid unintended conflict
+# with other gems. To explicitly export symbols you can use RUBY_FUNC_EXPORTED
+# selectively, or just remove this flag.
```

Comment:
Sorry for being the postman of annoying news, but our CI rejected this comment due to usage of `just`.

```
Failures:

  1) The library itself maintains language quality of sentences used in source code
     Failure/Error: expect(error_messages.compact).to be_well_formed


       Commands:
       $ git ls-files -z -- lib/bundler lib/bundler.rb
       # $? => 0
       lib/bundler/templates/newgem/ext/newgem/extconf-c.rb.tt:7 has 'just'. Avoid using these kinds of weak modifiers.
...
     # ./spec/bundler/quality_spec.rb:138:in `block (2 levels) in <top (required)>'
     # ./spec/bundler/spec_helper.rb:102:in `block (4 levels) in <top (required)>'
     # ./spec/bundler/spec_helper.rb:102:in `block (3 levels) in <top (required)>'
     # ./spec/bundler/support/helpers.rb:345:in `block in with_gem_path_as'
     # ./spec/bundler/support/helpers.rb:359:in `without_env_side_effects'
     # ./spec/bundler/support/helpers.rb:340:in `with_gem_path_as'
     # ./spec/bundler/spec_helper.rb:101:in `block (2 levels) in <top (required)>'

```

Reply:
Lol. Changed.